src/sqlfluff/rules/L019.py

```diff
@@ -123,7 +133,7 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
             # A comma preceded by a new line == a leading comma
             if context.segment.is_type("comma"):
                 last_seg = self._last_code_seg(context.raw_stack)
-                if last_seg.is_type("newline"):
+                if last_seg and last_seg.is_type("newline"):
```

Comment:
Checking that `last_seg` is not `None`. mypy wanted this, and this may have fixed a bug (I didn't study closely).